db/migrate/20201009142558_add_role_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddRoleToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :role, :string, null: false, default: ''
```

Comment:
Null is not allowed here but it's an empty string by default. I think it's preferable we just stop allowing User instances without a role.

Reply:
So having Users without a role will be handled when I add the validation you suggested, but I think it is still a good idea to have a default in this column. Maybe a 'none' if you don't want empty string?

Reply:
`none` seems like a reasonable option to me. It's not going to allow any permissions, but is something a little clearer if it were to ever somehow happen.